### DISCUSSION TITLE
Personal feed should have updates on issues also

### DISCUSSION THREAD

**Discussion by akshay-nm** [community/community#13628] [Other Feature Feedback, Questions, & Ideas]
I'd love to see updates on issues I have created. Maybe a little snippet showing 5 most recent updates.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

maybe they will be adding more filters to it
